dotcom-rendering/src/web/components/Island.tsx

```diff
@@ -9,13 +9,15 @@ interface HydrateProps {
 	clientOnly?: false;
 	placeholderHeight?: never;
 	children: JSX.Element;
+	expediteLoading?: boolean;
 }
 
 interface ClientOnlyProps {
 	deferUntil?: When;
```

Comment:
This prop started out life being called `when`. We moved to `deferUntil` because it better represented the options at the time but maybe now 'when' makes sense?

Reply:
I did think about having it in a single interface, e.g 
`when="expedited | loaded | idle | visible"` but I didn't think I could find something as concise as deferUntil - for example the default state (in this example loaded) didn't have an obvious name for me?

The other advantage of having it as its own prop is that it's likely to be used more thoughtfully - adding 'expediteLoading' should be a conscious and purposeful choice because if we use it too much it'll affect overall page speeds!

Would be open to new interface suggestions!!